app/extensions/logging/__init__.py

```diff
@@ -146,10 +151,10 @@ def delete_object(cls, logger, obj, msg='', *args, **kwargs):
     @classmethod
     def patch_object(cls, logger, obj, patch_args, *args, **kwargs):
 
-        msg = ''
+        msg = 'Patch: '
         for patch in patch_args:
 
-            msg = f"{patch['op']} {patch['field_name']}"
+            msg += f"{patch['op']} {patch['field_name']}"
```

Comment:
if there is no `value`, will this concatenation get kinda squashed together?  e.g. maybe it needs a space at front or end

Reply:
Done